src/@types/InteractionContext.d.ts

```diff
@@ -17,49 +17,34 @@ import type {
 } from 'discord.js';
 
 declare global {
-	interface BaseCommandContext {
+	interface InteractionContext {
 		/** Where the command was invoked. */
 		readonly source: 'guild' | 'dm';
 
-		/** The command invocation interaction. */
-		readonly interaction: CommandInteraction;
+		/** The triggering interaction. */
+		readonly interaction: RepliableInteraction;
```

Comment:
I think you may have forgotten to update your imports when you switched these out.
I still see `CommandInteraction` and I don't see `RepliableInteraction`.